mailsender/tool/src/webapp/content/js/mailsender.js

```diff
@@ -1,7 +1,75 @@
+var accumulatedFileSize = 0;
+var maxFileUploadSize;
```

Comment:
Hi @mylescarey2019 can you use let or const here vs var?

Reply:
suggestion completed